database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -16,9 +16,14 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->string('surname');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->string('phone')->unique()->nullable();
```

Comment:
The phone number defined as unique there, but you have not validated this correctly in above!